Tic Tac Toe.py

```diff
@@ -0,0 +1,175 @@
+import random
+
+dic = {'1':' ', '2':' ', '3':' ',
+       '4':' ', '5':' ', '6':' ',
+       '7':' ', '8':' ', '9':' '
+       }
+
+def print_template():
+    print('''===========================
+Welcome to Tic Tac Toe
+GAME RULES:
+Each player can place one mark (or stone) per turn on the 3x3 grid
+The WINNER is who succeeds in placing three of their marks in a
+* horizontal,
+* vertical or
+* diagonal row
+Let's start the game''')
+    print('------')
+    print(' | | ')
+    print('------')
+    print(' | | ')
+    print('------')
+    print(' | | ')
+    print('------')
+    print('='*27)
+
+def first_player():
+       lst = [0, 1]
+       random.shuffle(lst)
+       return lst
+
+
+def choose_sign(player1, player2):
+    while True:
+        if player1 == 1:
+            player = 'Player1'
+            choices = ['x', 'o']
+            sypher = input('Random choose {} as first player. {} please insert your sign: x or o \n'.format(player, player))
+            if sypher != choices[0] and sypher != choices[1]:
+                print('You have to insert x or o')
+                continue
+            else:
+                if sypher == 'x':
+                    return ['x', 'o']
+                else:
+                    return ['o', 'x']
+
+        elif player2 == 1:
+            player = 'Player2'
+            choices = ['x', 'o']
+            sypher = input('Random choose {} as first player. {} please insert your sign: x or o \n'.format(player, player))
+            if sypher != choices[0] and sypher != choices[1]:
+                print('You have to insert x or o ')
+                continue
+            else:
+                if sypher == 'x':
+                    return ['o', 'x']
+                else:
+                    return ['x', 'o']
```

Comment:
You should never copy code like this. This differs only with some variables. Try to write the logic only once and change variable values used.